### PR TITLE
Exercises part1

### PR DESCRIPTION
**Pull request by HuroNet**
Add the fuction cadenaALista de numeros

### CODE REVIEW

File: basic_excersises.py [hunk 1]
```diff
@@ -129,11 +120,27 @@ def string_for(lista):
...
+
+
+# print(stringToList("1,2,3,4,5"))
+
+# forma mas sencilla de hacer un cambio de
+# numero en cadena a lista
+#ejercico de entrevista
+
+def cadenaALista(cadena):
+    numeros = cadena.split(",")
```
Comment (marcos-gino95) [inline basic_excersises.py:141]: Change the camel case to snake case for python please

File: basic_excersises.py [hunk 2]
```diff
@@ -116,9 +109,7 @@ def string_inverse(lista):
 # print(string_inverse([1,2,43,9,73]))
 
 
-# con el bucle for cambio
-
-
+# con el bucle for cambio new change
 def string_for(lista):
```
Comment (marcos-gino95) [inline basic_excersises.py:113]: same here

File: basic_excersises.py [hunk 3]
```diff
@@ -95,13 +95,6 @@ def longitud(dato):
...
-def inversa2(cadena):
-    index = -1
-
-
-##inversa de una cadena
-
-
 def string_inverse(lista):
     inicio = 0
     fin = len(lista) - 1
```
Comment (marcos-gino95) [inline basic_excersises.py:100]: same here

**Review by marcos-gino95 — CHANGES_REQUESTED** [review]
LGTM
Just change the names

File: basic_excersises.py (hunk 2 above)
Comment (HuroNet) [reply basic_excersises.py:113]: done

**Review by marcos-gino95 — APPROVED** [review]
:100:

**Pull request merged**